warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PushdownLargeFieldedListsVisitor.java

```diff
@@ -58,12 +58,14 @@
     private String fstHdfsUri;
     private FileSystem fs;
     private Set<String> fields;
+    private long sourceLimit;
     
-    public PushdownLargeFieldedListsVisitor(ShardQueryConfiguration config, FileSystem fs, String fstHdfsUri, Set<String> fields) {
+    public PushdownLargeFieldedListsVisitor(ShardQueryConfiguration config, FileSystem fs, String fstHdfsUri, Set<String> fields, long sourceLimit) {
         this.config = config;
         this.fstHdfsUri = fstHdfsUri;
         this.fs = fs;
         this.fields = fields;
+        this.sourceLimit = sourceLimit;
```

Comment:
sourceLimit is never used. Either this should be used down below to collapse nodes that are over a source limit (if specified/nor -1) or this should be removed